api/views/author.py

```diff
@@ -255,3 +255,18 @@ def get_serializer_context(self):
         context = super(GetFriends, self).get_serializer_context()
         context['nested_queryset'] = queryset
         return context
+
+
+class QueryAuthors(APIView):
+    """
+    Queries the database for all authors
+
+    """
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, *args, **kwargs):
+        authors =  Author.objects.all()
+        serializer = CompactAuthorSerializer(authors, many=True)
+
+        return Response(serializer.data, status=status.HTTP_200_OK)
```

Comment:
I suspect this response would return `{ [ ... ] }`. Lets make it `{"authors": [ .. ] }`